services/src/main/java/io/grpc/services/HealthServiceImpl.java

```diff
@@ -29,13 +29,15 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.grpc.services.health;
+package io.grpc.services;
 
-import static io.grpc.services.health.HealthOuterClass.HealthCheckResponse;
-import static io.grpc.services.health.HealthOuterClass.HealthCheckResponse.ServingStatus;
+import static io.grpc.health.v1.HealthCheckResponse.ServingStatus;
```

Comment:
FYI, this can be a normal (non-static import). Doesn't matter which way you do it though.